uninstall.sh

```diff
@@ -46,10 +46,10 @@ function fancy_message() {
 	local MESSAGE="${2}"
 
 	case ${MESSAGE_TYPE} in
-		info) echo -e "[${GREEN}+${NC}] INFO: ${MESSAGE}" ;;
-		warn) echo -e "[${YELLOW}*${NC}] WARNING: ${MESSAGE}" ;;
-		error) echo -e "[${RED}!${NC}] ERROR: ${MESSAGE}" ;;
-		*) echo -e "[?] UNKNOWN: ${MESSAGE}" ;;
+		info) echo -e "[${BGreen}+${NC}] INFO: ${MESSAGE}" ;;
```

Comment:
Fix variable names for the colors color

Reply:
They aren't defined in this script tho
![image](https://user-images.githubusercontent.com/65723952/177016496-3dfe03e8-bf65-4829-90e5-1dbb3e302788.png)